public_html/administrator/components/com_jed/views/review/tmpl/edit.php

```diff
@@ -0,0 +1,159 @@
+<?php
+/**
+ * @package    JED
+ *
+ * @copyright  Copyright (C) 2005 - 2019 Open Source Matters, Inc. All rights reserved.
+ * @license    GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+defined('_JEXEC') or die;
+
+use Joomla\CMS\HTML\HTMLHelper;
+use Joomla\CMS\Language\Text;
+
+/** @var JedViewReview $this */
+
+JHtml::_('behavior.formvalidator');
+JHtml::_('formbehavior.chosen', 'select');
```

Comment:
Please do not use `select`, if you have fields that need chosen you can set the class `advancedSelect` on the dropdowns that need chosen.